autogpt_platform/frontend/src/components/nav/MarketPopup.tsx

```diff
@@ -7,7 +7,15 @@ interface MarketPopupProps extends ButtonHTMLAttributes<HTMLButtonElement> {
 
 export default function MarketPopup({
   className = "",
-  marketplaceUrl = "http://platform.agpt.co/marketplace",
+  marketplaceUrl = (() => {
+    if (process.env.APP_ENV === "prod") {
+      return "https://production-marketplace-url.com";
+    } else if (process.env.APP_ENV === "dev") {
+      return "https://dev-builder.agpt.co/marketplace";
+    } else {
+      return "http://localhost:3000/marketplace";
+    }
+  })(),
```

Comment:
This isn't actually accessible on the frontend in testing :/ i think we will need a different way to handle this

Reply:
I think you want to use the build variables